dockerfiles/Dockerfile-mina-daemon

```diff
@@ -58,7 +58,7 @@ RUN apt-get update --quiet --yes \
 RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list \
   && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add - \
   && apt-get update --quiet --yes \
-  && apt-get install --quiet --yes --no-install-recommends google-cloud-sdk kubectl google-cloud-sdk-gke-gcloud-auth-plugin \
+  && apt-get install --quiet --yes --no-install-recommends google-cloud-sdk=462.0.1-0 kubectl=1:462.0.1-0 google-cloud-sdk-gke-gcloud-auth-plugin=462.0.1-0 \
```

Comment:
Can we comment here why we are sticking to particular commits?

Reply:
This was a hack to get around the google version issue that was blocking docker builds last week. I put this hack in here before the actual fix was merged into upstream branches. I'll try removing now and see if the upstream fixes worked or not.